main/main.cpp

```diff
@@ -1273,7 +1273,7 @@ bool Main::start() {
 	bool hasicon = false;
 	String doc_tool;
 	List<String> removal_docs;
-	bool doc_base = true;
+	bool doc_base ATTRIBUTE_UNUSED = true;
```

Comment:
Same thing with this one, should be enclosed in `#ifdef TOOLS_ENABLED` (including the parsing of `--no-docbase`).